src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -518,21 +524,25 @@ public function handleException($exception, array $error = null)
         $handlerException = null;
 
         if (($this->loggedErrors & $type) || $exception instanceof FatalThrowableError) {
+            if (false !== strpos($message = $exception->getMessage(), "class@anonymous\0")) {
+                ($message = new FlattenException())->setMessage($message);
```

Comment:
naming the temporary variable `$message` here looks weird, as it does not contain the message.

Reply:
Fixed by making `FlattenException` fluent.